packages/website/pages/_document.tsx

```diff
@@ -20,6 +35,7 @@ export default class MyDocument extends Document<{
             data-emotion-css={this.props.ids.join(' ')}
             dangerouslySetInnerHTML={{ __html: this.props.css }}
           />
+          <script dangerouslySetInnerHTML={{ __html: this.renderSnipet() }} />
```

Comment:
can we do `async` here?

Reply:
I'd rather not render `<script>` at all if `__html` is empty

Reply:
sure, make sense.